WindowsClient/_Data/_Actions/Actions.cs

```diff
@@ -18,10 +18,8 @@ class Actions
 
         public static void ShowFileContent(string fileName)
         {
-            string folder = Path.Combine(Environment.GetFolderPath(
-    Environment.SpecialFolder.ApplicationData), ".PIM");
             // Filename  
-            string fullPath = folder + "/" + fileName;
+            string fullPath = DirectoryDialog.defaultRepo + "/" + fileName;
```

Comment:
replace this line with $"{}" for future possible implementions and code readability